pkg/testrunner/result/summary-poster_test.go

```diff
@@ -0,0 +1,58 @@
+package result
+
+import (
+	tmv1beta1 "github.com/gardener/test-infra/pkg/apis/testmachinery/v1beta1"
+	"github.com/gardener/test-infra/pkg/testmachinery/metadata"
+	"github.com/gardener/test-infra/pkg/testrunner"
+	"github.com/gardener/test-infra/pkg/util"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("summary-poster", func() {
+
+	It("should render summary with 1 and 2 results for the same line", func() {
```

Comment:
could this be split into 2 separate tests? If the fails, it would be more obvious where the problem is located.